### PR TITLE
Migrate Lidar from STM32 to Raspberry Pi

### PR DESCRIPTION
**Pull request by ecourtois**

### CODE REVIEW

**Review by gdoffe — CHANGES_REQUESTED** [review]
Good job ! A new important step !

A small fix needed before approval.

File: platforms/pegasus/trace_utils.cpp [hunk 1]
```diff
@@ -7,16 +7,37 @@
 #include "platform.hpp"
 
 /* Periodic task */
-#define TASK_PERIOD_USEC    (500 * US_PER_MS)
+#define TASK_PERIOD_USEC_POSE    (50 * US_PER_MS)
+#define TASK_PERIOD_USEC_STATE    (200 * US_PER_MS)
```
[the remark this answers is not in the document]
Comment (ecourtois) [reply platforms/pegasus/trace_utils.cpp:11]: ZTIMER_USEC is used in all other threads, Is there a reason to use MSEC here?

[the remark this answers is not in the document]
Comment (gdoffe) [reply platforms/pegasus/trace_utils.cpp:11]: Yes we are using milliseconds => MSEC.
This should be done this way in all other threads.

I will fix all remaining USEC threads soon, however as you are modifying this one, good occasion to use MSEC now.

File: platforms/pf_test/trace_utils.cpp [hunk 2]
```diff
@@ -7,16 +7,37 @@
 #include "platform.hpp"
 
 /* Periodic task */
-#define TASK_PERIOD_USEC    (500 * US_PER_MS)
+#define TASK_PERIOD_USEC_POSE    (50 * US_PER_MS)
+#define TASK_PERIOD_USEC_STATE    (200 * US_PER_MS)
 
 /* Thread stack */
-static char trace_thread_stack[THREAD_STACKSIZE_MEDIUM];
+static char pose_thread_stack[THREAD_STACKSIZE_SMALL];
```
[the remark this answers is not in the document]
Comment (gdoffe) [reply platforms/pf_test/trace_utils.cpp:14]: I'm ok with this!

File: platforms/pegasus/trace_utils.cpp (hunk 1 above)
[the remark this answers is not in the document]
Comment (ecourtois) [reply platforms/pegasus/trace_utils.cpp]: Ok! Done!

**Review by gdoffe — APPROVED** [review]
LGTM !

**Pull request merged**